src/Telemetry/AzureDevOpsClient/AzureDevOpsClient.cs

```diff
@@ -167,14 +167,17 @@ public async Task<WorkItem> CreateRcaWorkItem(string project, string title, Canc
         using var response = await _httpClient.SendAsync(request, HttpCompletionOption.ResponseHeadersRead, cancellationToken);
         using Stream logStream = await response.Content.ReadAsStreamAsync(cancellationToken);
         using StreamReader reader = new StreamReader(logStream);
-        string line;
-        while ((line = reader.ReadLine()) != null)
+        // The OneES regex needs two lines to correctly find the match
+        string firstLine = reader.ReadLine();
+        string secondLine;
+        while ((secondLine = reader.ReadLine()) != null)
```

Comment:
This works, but is not perfect from the performance perspective. Practically, you only want to check for the Image regexp on the second line when you match the SKU regexp on the first line. Concatenating the two lines together might be less performant for huge log files. I'm not asking you to change this unless we are processing huge (GBs) logs.

Reply:
You're right, I improved the logic a bit